z3-sys/build.rs

```diff
@@ -31,16 +60,26 @@ fn main() {
         "symbol_kind",
     ] {
         let mut enum_bindings = bindgen::Builder::default()
-            .header(&header)
+            .header(header)
             .parse_callbacks(Box::new(bindgen::CargoCallbacks))
             .generate_comments(false)
             .rustified_enum(format!("Z3_{}", x))
             .allowlist_type(format!("Z3_{}", x));
-        if env::var("TARGET").unwrap() == "wasm32-unknown-emscripten" {
-            enum_bindings = enum_bindings.clang_arg(format!(
-                "--sysroot={}/upstream/emscripten/cache/sysroot",
-                env::var("EMSDK").expect("$EMSDK env var missing. Is emscripten installed?")
-            ));
+        let target = env::var("TARGET").unwrap();
+        let wasm32 = target.starts_with("wasm32-unknown");
+        let wasm32_emscripten = target == "wasm32-unknown-emscripten";
+        if wasm32 {
+            let sysroot = env::var("EMSDK")
```

Comment:
It seems that sometimes the cache may be missing for fresh emscripten installation. So I'm not sure whether we should add some notices here.